packages/cadl-ranch-specs/http/models/property-types/mockapi.ts

```diff
@@ -63,7 +69,7 @@ const enumMock = createMockApis("enum", "ValueOne");
 Scenarios.Models_Property_Types_Enum_get = passOnSuccess(enumMock.get);
 Scenarios.Models_Property_Types_Enum_put = passOnSuccess(enumMock.put);
 
-const extensibleEnumMock = createMockApis("extensible-enum", "UnknownValue");
+const extensibleEnumMock = createMockApis("extensible-enum", "ValueOne");
```

Comment:
extensible-enum means client/service can use any value (I assume the "UnknownValue" is for this case).

Reply:
[ExtensibleEnumProperty](https://github.com/Azure/cadl-ranch/blob/9e5ddbeda8413655341e92392ea288ccc67f2064/packages/cadl-ranch-specs/http/models/property-types/main.cadl#L116) has [InnerExtensibleEnum](https://github.com/Azure/cadl-ranch/blob/9e5ddbeda8413655341e92392ea288ccc67f2064/packages/cadl-ranch-specs/http/models/property-types/main.cadl#L112-L113) inside whose allowed values are only ValueOne and ValueTwo. I'm not aware we could accept any values. @archerzz do you know?